oauth2client/flask_util.py

```diff
@@ -15,10 +15,17 @@
 """Utilities for the Flask web framework
 
 Provides a Flask extension that makes using OAuth2 web server flow easier.
-The extension includes views that handle the entire auth flow and a @required
-decorator to automatically ensure that user credentials are available.
+The extension includes views that handle the entire auth flow and a
+``@required`` decorator to automatically ensure that user credentials are
+available.
 
-To configure::
+
+Configuration
+=============
+
+To configure, you'll need a set of OAuth2 client ID from the
+`Google Developer's Console <https://console.developers.google.com/project/_/\
```

Comment:
@soltanmm Looks like the link actually gets created as expect. Maybe Sphinx got smart?

![screen_shot_025](https://cloud.githubusercontent.com/assets/520669/9285615/827e9b98-429c-11e5-80fd-4716663f54c7.png)